File: lectures/OOP.js
class City { // Define your class with "class nameOfClass"
    constructor(cityName, countryName, currentPopulation) { // constructor is equivalent to "__init__"
        this.city = cityName;
        this.country = countryName;
        this.population = currentPopulation;
    }

    // Methods for your class go inside here.
    takeCensus(newPopulation) {
        this.population = newPopulation;
    }
}

var seattle = new City("Seattle","United States",750000); // Creating an instance of a class
console.log(seattle.population);
seattle.takeCensus(800000);
console.log(seattle.population);

File: lectures/OOP.py
class City():
    def __init__(self,city_name,country_name, population_count):
        self.name = city_name
        self.country = country_name
        self.population = population_count

    def take_census(self,new_count):
        self.population = new_count

seattle = City("Seattle","United States",750000)
print(seattle.population)
print(seattle.country)
seattle.take_census(800000)
print(seattle.population)

